packages/pebble-web/src/components/Select.tsx

```diff
@@ -48,9 +49,15 @@ function Select<OptionType>(props: SelectProps<OptionType>) {
         })}
         onOutsideClick={isOpen => onDropdownToggle(isOpen)}
         labelComponent={({ toggleDropdown, isOpen }) => {
-          const chevron = cx(chevronStyle, "pi", "pi-arrow-drop-down", {
-            __pebble__select__open: isOpen
-          });
+          const chevron = cx(
+            chevronStyle,
+            "pi",
+            "pi-arrow-drop-down",
+            {
+              __pebble__select__open: isOpen
```

Comment:
Where are you getting these class names , i have seen this in the chevron style as well